pkg/util/storage_network.go

```diff
@@ -0,0 +1,17 @@
+package util
+
+const (
+	// Rancher monitoring
+	CattleMonitoringSystemNamespace = "cattle-monitoring-system"
+
+	RancherMonitoringPrometheus = "rancher-monitoring-prometheus"
+
+	RancherMonitoringAlertmanager = "rancher-monitoring-alertmanager"
+
+	RancherMonitoring = "rancher-monitoring"
+
+	RancherMonitoringGrafana = "rancher-monitoring-grafana"
+
+	// VM import controller
+	HarvesterVMImportController = "vm-import-controller-harvester-vm-import-controller"
```

Comment:
Typo here?
Should this string be `harvester-vm-import-controller`?

Reply:
No, it's the actual name for the depolyment